controllers/extendeddaemonsetreplicaset/strategy/rollingupdate.go

```diff
@@ -117,8 +125,17 @@ func ManageDeployment(client runtimeclient.Client, params *Parameters) (*Result,
 	nbPodToCreateWithConstraint := utils.MinInt(nbPodToCreate, len(allPodToCreate))
 	params.Logger.V(1).Info("Pods actions with limits", "nbPodToDelete", nbPodToDelete, "nbPodToCreate", nbPodToCreate, "nbPodToDeleteWithConstraint", nbPodToDeleteWithConstraint, "nbPodToCreateWithConstraint", nbPodToCreateWithConstraint)
```

Comment:
could you update the log line with the `frozen` and `paused` info?

Reply:
good point! done